tools/start_server.py

```diff
@@ -232,32 +303,29 @@ def install_app(self, _app_id, _package_url):
       return RPCService.gen_error_msg('Failed to unzip app')
 
     # Start file server
-    print('\033[1mStarting File Server\033[0m')
-
-    fs = self.start_app_file_server(app_storage_folder)
+    print('\033[1mAdding app to File Server\033[0m')
+    file_server_info = self.add_app_to_file_server(app_storage_folder)
     WebEngineManager.webengine_apps[_app_id] = {
       "policyAppID": _app_id,
-      "fileServerProcess": fs['process'],
-      "fileServerURL": fs['url']
+      "appURL": file_server_info['url'],
+      "appKey": file_server_info['key']
     }
 
     return {
-      'success': True,
-      'params': {
-        'appUrl': fs['url']
+      "success": True,
+      "params": {
+        "appUrl": file_server_info['url']
       }
     }
 
-  def start_app_file_server(self, _app_storage_folder):
-    port = WebEngineManager.next_available_port
+  def add_app_to_file_server(self, _app_storage_folder):
     secret_key = str(uuid.uuid4())
-    print('\033[2mPort is %s, secret key is %s\033[0m' % (port, secret_key))
-
-    process = subprocess.Popen(['python3', '../../tools/file_server.py', str(self.WEBENGINE_HOST), str(port), secret_key], cwd=_app_storage_folder)
-    time.sleep(1)
-
-    WebEngineManager.next_available_port += 1
-    return {'process': process, 'url': 'http://%s:%s/%s/' % (WebEngineManager.WEBENGINE_REMOTE_HOST, port, secret_key)}
+    print('\033[2mSecret key is %s\033[0m' % (secret_key))
```

Comment:
i think this print can be removed

Reply:
Fixed in https://github.com/smartdevicelink/generic_hmi/pull/273/commits/7b8af7053201d5054d86aafe6805c288ed83f666